plugins/synapticsmst/synapticsmst.quirk

```diff
@@ -15,7 +15,7 @@
 # To override this behavior add the custom flag "skip-restart"
 #
 
-[SynapticsMSTBoardID=0]
+[SynapticsMSTBoardID=2]
```

Comment:
Is this intended change?

Reply:
I think this should be only pending question.  Is this intended change? (Eg should EVB be board ID 2 for new board address reading procedure)?

If so, there are a few other places in the code that treat EVB specially and should be adjusted too.